packages/components/src/styles/interactive/interactive.stories.ts

```diff
@@ -64,88 +36,41 @@ export const Default = {
 };
 
 /**
- * Use the `disabled` class to make an interactive element look disabled. This works as well when setting an `disabled` attribute on the element.
+ * Use the `sd-interactive--reset` class to reset the default browser styles of e. g. a button.
```

Comment:
Can we move the "Reset" section to the end?

Reply:
Updated